service/spi/src/main/java/org/kaazing/gateway/service/DefaultLongMonitoringCounterStub.java

```diff
@@ -19,9 +19,9 @@
  * under the License.
```

Comment:
This file should not have been moved out of gateway.management. It does not belong in gateway.service because that represents the SPI for creating services. Services should not need to use implementation classes like this one.